sdk/identity/identity/test/manual-integration/AzureServiceFabric/INSTRUCTIONS.md

```diff
@@ -0,0 +1,257 @@
+# Testing azure-identity in Service Fabric (in TS)
+
+Setup for a Service Fabric cluster and two apps, used for testing managed identity using Azure.Identity.
+
+The `sfmitestsystem` and `sfmitestuser` directories contain mock applications that use Azure.Identity's `ServiceFabricCredential` to request and verify Key Vault access tokens. The former application uses a system-assigned managed identity to do so, and the latter application uses a user-assigned managed identity.
+
+The `arm-templates` directory contains Azure resource templates for creating these applications as well as a Service Fabric cluster to host them. The cluster template also deploys other resources that are necessary for running a cluster: a load balancer, public IP address, virtual machine scale set, virtual network, and two storage accounts.
+
+### Environment requirements
+
+> **Note:** All Azure resources used in the sample should be in the same region & resource group.
+
+- This sample requires access to an Azure subscription and required privileges to create resources.
+- [An SSH key pair is required.](https://docs.microsoft.com/azure/virtual-machines/linux/mac-create-ssh-keys#provide-an-ssh-public-key-when-deploying-a-vm)
+- [Azure CLI is used to deploy resources and applications.](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Docker is needed to build and push the sample containerized services. Docker should be using Linux containers for building the application images that are provided.
+
+### Clone this repository
+
+From a command prompt window, run
+
+```
+git clone https://github.com/Azure/azure-sdk-for-js --single-branch --branch master --depth 1
+cd azure-sdk-for-js/sdk/identity/identity/test/manual-integration/AzureServiceFabric
+```
+
+### Sections
+
+- [Resource setup](#set-up-resources)
+- [Application deployment](#set-up-and-deploy-the-applications)
+- [Test validation](#run-the-tests)
+
+## Set Up Resources
+
+You can skip to [Set Up and Deploy the Applications](#set-up-and-deploy-the-applications) if you have an existing Service Fabric cluster, key vault, storage account, container registry, and managed identity named "AdminUser".
+
+### Create a resource group
+
+From a command prompt window, run:
+
+```
+az login
+az group create -n $RESOURCE_GROUP --location $LOCATION --subscription $SUBSCRIPTION_NAME
```

Comment:
This helped me, so I think it’s worth adding it:

```
az account set --subscription $SUBSCRIPTION_NAME
```

Reply:
```suggestion
az group create -n $RESOURCE_GROUP --location $LOCATION --subscription $SUBSCRIPTION_NAME
az account set --subscription $SUBSCRIPTION_NAME

```